views/kubernetes-cluster/create.erb

```diff
@@ -0,0 +1,29 @@
+<% @page_title = "Create Kubernetes Cluster" %>
+
+<%== render(
+  "components/page_header",
+  locals: {
+    breadcrumbs: [
+      %w[Projects /project],
+      [@project_data[:name], @project_data[:path]],
+      ["Kubernetes Clusters", "#{@project_data[:path]}/kubernetes-cluster"],
+      %w[Create #]
+    ]
+  }
```

Comment:
```suggestion
<%== part(
  "components/page_header",
  breadcrumbs: [
    %w[Projects /project],
    [@project_data[:name], @project_data[:path]],
    ["Kubernetes Clusters", "#{@project_data[:path]}/kubernetes-cluster"],
    %w[Create #]
  ]
```

I recently added a commit to use `part` instead of `render` with `:locals`.  Above is an example of such a change.  Please change all other `render` calls with `:locals` to use `part`.

Reply:
Wow :D I will need to do a bunch of rebases to get there